api/config.py

```diff
@@ -41,6 +41,14 @@ class DevelopmentConfig(BaseConfig):
     TESTING = True
     ENV = 'development'
     ID_LIST_FILE = BaseConfig.ROOT_DIR / 'cards/test_users.json'
+    START_DATETIME = datetime(2019, 1, 1, 0, 0, 0, tzinfo=BaseConfig.TIMEZONE)
```

Comment:
ちょっと思ったのが、これだと「常時応募受付期間である」ということが分かりづらさある(うちらの代はいいのだがその後なんども引き継いだ後)ので注釈コメント入れた方がいいかも。
こんな感じか？(時間の呼び方とかは前決めた気がするが探し出せなかった)

```python
# Because of those configs, the system is always in　application period.
# Please modify 'END_DATETIME'/'START_DATETIME' to change the period.
# ex: before Sakuten, electing time
START_DATETIME = datetime(2019, 1, 1, 0, 0, 0, tzinfo=BaseConfig.TIMEZONE)
...
```